File: Player/src/main/java/ru/testsimpleapps/coloraudioplayer/ui/dialogs/PlaylistRenameDialog.java
package ru.testsimpleapps.coloraudioplayer.ui.dialogs;

import android.content.Context;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import ru.testsimpleapps.coloraudioplayer.R;
import ru.testsimpleapps.coloraudioplayer.ui.activities.MainActivity;

public class PlaylistRenameDialog
        extends BaseDialog
        implements View.OnClickListener {

    private final Context context;
    private Button renameButton;
    private Button cancelButton;
    private EditText editText;

    public PlaylistRenameDialog(Context context) {
        super(context);
        this.context = context;
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.playlist_rename_dialog);

        renameButton = (Button) findViewById(R.id.playlistRename_create);
        cancelButton = (Button) findViewById(R.id.playlistRename_cancel);
        editText = (EditText) findViewById(R.id.playlistRename_edit);
        renameButton.setOnClickListener(this);
        cancelButton.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.playlistRename_create:
                if (!editText.getText().toString().trim().equals("")) {
                    //CursorTool.renamePlaylist(context.getContentResolver(), App.getAppContext().getPlaylistId(), editText.getText().toString());

                    dismiss();
                    MainActivity.showInfoMessage(context.getResources().getString(R.string.playlist_rename_done));
                } else
                    MainActivity.showInfoMessage(context.getResources().getString(R.string.playlist_rename_wrong_name));
                break;
            case R.id.playlistRename_cancel:
                dismiss();
                break;
        }
    }
}

File: player/src/main/java/ru/testsimpleapps/coloraudioplayer/ui/dialogs/PlaylistRenameDialog.kt
package ru.testsimpleapps.coloraudioplayer.ui.dialogs

import android.content.Context
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText

import ru.testsimpleapps.coloraudioplayer.R

class PlaylistRenameDialog(context: Context) : BaseDialog(context), View.OnClickListener {

    private var renameButton: Button? = null
    private var cancelButton: Button? = null
    private var editText: EditText? = null

    override fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.dialog_playlist_rename)

        renameButton = findViewById<View>(R.id.playlistRename_create) as Button
        cancelButton = findViewById<View>(R.id.playlistRename_cancel) as Button
        editText = findViewById<View>(R.id.playlistRename_edit) as EditText
        renameButton!!.setOnClickListener(this)
        cancelButton!!.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.playlistRename_create -> {
                if (editText!!.text.toString().trim { it <= ' ' } != "") {
                    //CursorTool.renamePlaylist(instance.getContentResolver(), App.getAppContext().getPlaylistId(), editText.getText().toString());

                    dismiss()

                }

                dismiss()
            }
            R.id.playlistRename_cancel -> dismiss()
        }
    }
}
